docs/source/gateway/index.rst

```diff
@@ -240,22 +240,30 @@ With the rapid development of LLMs, there is no guarantee that this list will be
 below can be used as a helpful guide when configuring a given route for any newly released model types as they become available with a given provider.
 ``N/A`` means that the provider currently doesn't support the route type.
 
-+--------------------+--------------------------+------------------+-----------------------------+--------------------------+
-| Route Type         | OpenAI                   | Anthropic        | Cohere                      | Azure OpenAI             |
-+====================+==========================+==================+=============================+==========================+
-| llm/v1/completions | - gpt-3.5-turbo          | - claude-1       | - command                   | - text-davinci-003       |
-|                    | - gpt-4                  | - claude-1.3-100k| - command-light-nightly     | - gpt-35-turbo           |
-+--------------------+--------------------------+------------------+-----------------------------+--------------------------+
-| llm/v1/chat        | - gpt-3.5-turbo          | N/A              | N/A                         | - gpt-35-turbo           |
-|                    | - gpt-4                  |                  |                             | - gpt-4                  |
-+--------------------+--------------------------+------------------+-----------------------------+--------------------------+
-| llm/v1/embeddings  | - text-embedding-ada-002 | N/A              | - embed-english-v2.0        | - text-embedding-ada-002 |
-|                    |                          |                  | - embed-multilingual-v2.0   |                          |
-+--------------------+--------------------------+------------------+-----------------------------+--------------------------+
++--------------------+--------------------------+------------------+-----------------------------+--------------------------+-------------------------+
+| Route Type         | OpenAI                   | Anthropic        | Cohere                      | Azure OpenAI             | MLflow                  |
++====================+==========================+==================+=============================+==========================+=========================+
+| llm/v1/completions | - gpt-3.5-turbo          | - claude-1       | - command                   | - text-davinci-003       | - transformers*         |
```

Comment:
I think we should call this "MLflow served models" instead of transformers. This can also be used with pyfunc, for example.